src/i18n/ge.js

```diff
@@ -0,0 +1,46 @@
+(function(jsGrid) {
+
+    jsGrid.locales.ge = {
```

Comment:
If we look at standard ISO 639-1 locale names https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes we see that it's `ka`. So I'd rather go with the standard name, if you don't mind. Thank you!